node/buffer.ts

```diff
@@ -245,6 +245,16 @@ export class Buffer extends Uint8Array {
     );
   }
 
+  /**
+   * Reads byteLength number of bytes from buf at the specified offset and interprets
+   * the result as an unsigned big-endian integer supporting up to 32 bits of accuracy.
+   */
+  readUIntBE(offset = 0, byteLength: number) {
+    if (byteLength === 4) return this.readUInt32BE(offset);
+    if (byteLength === 2) return this.readUInt16BE(offset);
+    if (byteLength === 1) return this.readUInt8(offset);
```

Comment:
Node.js version supports 1, 2, 3, 4, 5, 6 for byteLength, not only 1, 2, 4

Maybe let's use `DataView` and `getBigUint64()` method to get up to 6 bytes Uint, and then cast them to the number.

We also need to check the range of byteLength.

Reply:
@kt3k I just add boundsError in order to throw error if The value of "byteLength" is out of range. The probleme is that DataView does not support `getUint64` and if we use `getBigUint64` the result will be different from `node js ` result. 
I suggest to make another PR to support ` 5, 6` byteLength